app/controllers/spree/admin/tax_rates_controller.rb

```diff
@@ -35,10 +21,18 @@ def associated_adjustments?
         Spree::Adjustment.where(originator: @tax_rate).exists?
       end
 
-      def clone_tax_rate(tax_rate)
-        cloned_rate = tax_rate.deep_dup
-        cloned_rate.calculator = tax_rate.calculator.deep_dup
-        cloned_rate
+      def transition_tax_rate
+        if transition_rate!
```

Comment:
I would have found it simpler to call `updater.transition_rate!` here instead of a delegation but it's also good as is.